src/mapeo-manager.js

```diff
@@ -92,7 +92,12 @@ export class MapeoManager extends TypedEmitter {
    * @param {string} opts.projectMigrationsFolder path for drizzle migrations folder for project database
    * @param {string} opts.clientMigrationsFolder path for drizzle migrations folder for client database
    * @param {string | import('./types.js').CoreStorage} opts.coreStorage Folder for hypercore storage or a function that returns a RandomAccessStorage instance
-   * @param {{ port?: number, logger: import('fastify').FastifyServerOptions['logger'] }} [opts.mediaServerOpts]
+   * @param {Object} [opts.mediaServerOpts]
+   * @param {import('fastify').FastifyServerOptions['logger']} [opts.mediaServerOpts.logger]
+   * @param {Object} [opts.mediaServerOpts.mapsOpts]
+   * @param {string } [opts.mediaServerOpts.mapsOpts.staticMapsFolder]
```

Comment:
not in love with the nesting but wanted the flexibility to add more options under the maps scope in the future